### DISCUSSION TITLE
time lapse mode and continuous motion detection

### DISCUSSION THREAD

**Discussion by Fougl** [gopro/labs#474] [General]
Hi, I am using boot comand
!MTUSB=1!MBOOT="!Ltest"!SAVEtest=!SM3D4H5!R

However, it is not working continuously with time lapse. In fact it only triggers the recording, and then I have to manually press the shutter to stop recording (unless setting duration). Im using Gopro Hero11 Timelapse 0.5s 5.3k video. Tried using full hd, same result. Also with the photo mode.

In fact, the above qr works continuously only with time lapse photo format with 5s interval. If I use 2, 1 or 0.5s interval it doesnt work even in photo. In video doesnt work at all. But this doesnt make any sense

This is what I get when trying to do the !MTUSB=1!MBOOT="!Ltest"!SAVEtest=!SM3D4H5!R in timelapse video format. I have triggered the video with my hand, it starts recording correctly, but then the it shoots indefinitely until I stop it by pressing the record shutter button
![image](https://github.com/gopro/labs/assets/21178929/63e7c774-a8ee-4a90-a9c9-f2bf9fa2515b)

compared to the video 5.3k 24fps wide, where everything works as expected
![image](https://github.com/gopro/labs/assets/21178929/5a3218e4-f81c-4705-845f-43c7cbad44ba)

**Comment by dnewman-gpsw** [comment]
I have confirmed your findings.  The good news is it works fine in the upcoming release that I'm testing now.  It seem motion detection was improved.  Unfortunately I haven't found a work around in the current firmware.

**Reply by Fougl** [reply]
thanks. you are fast as always. amazing job

any estimation on when the new firmware will be realeased?

**Comment by Fougl** [comment]
Hi @dnewman-gpsw , i just want so send a reminder about this feature. I am really dependant on this one. I just dont want you to forget about it. Also about this one #187 

Will you be able to do it?

**Reply by dnewman-gpsw** [reply]
I think so.  The build is done, awaiting final release in a week or so.

**Comment by Fougl** [comment]
Seems it works perfectly on the new v2.12.70
WIll this work on Hero 11 Mini as well?
Thanks a lot

**Reply by dnewman-gpsw** [reply]
Mini might get these updates, although was already running some of the newer fixes.  Test it on Mini.

**Comment by Fougl** [comment]
Hi, I tested on Mini, it does not work properly on LABS_MINI11_02_30_70. Does it work on any older firmware? Would love to see it on mini

**Reply by dnewman-gpsw** [reply]
Mini doesn't have photo support, but motion detection with video using !MBOOT="!Ltest"!SAVEtest=!SM3D4H5!R is working here on Mini.

**Reply by Fougl** [reply]
I am talking about time lapse with motion detection... Motion detection is working, but not in video time lapse mode

**Reply by dnewman-gpsw** [reply]
That will likely be fixed if there Labs update, I think TLV is similar enough on Mini.

**Reply by dnewman-gpsw** [reply]
P.S. It current works in you use TimeWarp.  This worked mTW!SM3D4H5!R

**Reply by Fougl** [reply]
Yeah but I need time lapse. Ok I will be waiting if there will be Labs update

**Reply by dnewman-gpsw** [reply]
What interval do you need in Timelaspe Video, as TimeWarp can be 0.5 and 1s?

**Comment by Fougl** [comment]
I need 0.5s. Ah i didnt know what exactly is time warp. will test it. It can work the same right if my camera is fixed?

**Reply by dnewman-gpsw** [reply]
Almost the same yes, mTWp2 is timewarp at 2 fps. TW uses a tad more power, that is the only downside.

**Reply by Fougl** [reply]
yeah works. thanks. but i need time lapse 0.5s, which is basically 2fps. Can time warp be 2pfs?

**Reply by dnewman-gpsw** [reply]
It is 2 fps, typo.  0.5 equiv mode.
